### ISSUE TITLE
Move or swap entire stack

### ISSUE THREAD

**Issue by kiryph** [koekeishiya/yabai#1351]
When I have created a stack with several windows, can I move or swap the entire stack?

**Comment by koekeishiya** [comment]
Yes, use the usual window swap command. Warp however will take a window out of a stack, rather than operating on the stack itself.

**Comment by kiryph** [comment]
> Yes, use the usual window swap command.

Thanks for pointing this out. So my question then would be:

Can I swap a single window in a stack with a different window?

> Warp however will take a window out of a stack, rather than operating on the stack itself.

Is there interest in extending Yabai to support warping/moving entire stacks?

**Comment by koekeishiya** [comment]
> Can I swap a single window in a stack with a different window?

I don't think this is currently possible, no.

> Is there interest in extending Yabai to support warping/moving entire stacks?

This can be done indirectly already; if you have a window A to the west of a stack B, you can modify the dimensions of stack B by warping window A onto it. If you want the complete layout to be the inverse, you can first swap position of window A with entire stack B, then warp window A onto stack B as before.
